.travis.yml

```diff
@@ -5,4 +5,3 @@ install:
   - curl -O https://storage.googleapis.com/appengine-sdks/featured/google_appengine_1.9.17.zip && unzip -q google_appengine_1.9.17.zip
   - pip install -r requirements-dev.txt
 script: nosetests -w appengine --with-gae --exclude-dir={myria,raco,ply,networkx,sqlalchemy} --gae-lib-root=google_appengine
-sudo: false
```

Comment:
Is this necessary? In general, the [new Travis container-based infrastructure](http://docs.travis-ci.com/user/workers/container-based-infrastructure/#Routing-your-build-to-container-based-infrastructure) speeds up builds.